test/shared/commonFunctionsForTests.sh

```diff
@@ -736,3 +737,45 @@ function gchp_enable_or_disable_bootstrap() {
         fi
     done
 }
+
+
+function get_site_name() {
+    #========================================================================
+    # Returns the site name based on the node name (i.e. uname -n)
+    #========================================================================
+    nodeName=$(uname -n)
+    [[ "${nodeName}" =~ 'harvard.edu' ]] && echo "CANNON"
+    [[ "${nodeName}" =~ 'wustl.edu'   ]] && echo "COMPUTE1"
+    return 0
+}
+
+
+function get_sed_partition_cmd_from_site() {
+    #========================================================================
+    # Returns the sed command to replace the partition name in scripts
+    #
+    # 1st argument: Site name (cannon|compute1)
+    #========================================================================
+    [[ "X${1}" == "XCANNON"   ]] && p="sapphire,huce_cascade,seas_compute,shared"
+    [[ "X${1}" == "XCOMPUTE1" ]] && p="rvmartin"
+    echo "s/REQUESTED_PARTITION/${p}/"
+    return 0
+}
+
+
+function get_default_gcc_env_file() {
+    #========================================================================
+    # Returns the default environment file for GEOS-Chem Classic
+    #========================================================================
+    echo "/n/home09/ryantosca/repos/cannon-env/envs/gnu10/gcclassic.rocky+gnu10.minimal.env"
```

Comment:
I'm wondering if it would be better to put the env files in for example `run/GCClassic/runScriptSamples/operational_examples/harvard_cannon` rather than pointing to anyone's home directory. I notice there are env files for other clusters in that location.

Reply:
We can definitely do this.  I'll push.

Reply:
@msulprizio: Now fixed in commits 920b062 and 2ea36ea